controllers/build-tool-detector.go

```diff
@@ -12,15 +12,21 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-
 	"github.com/fabric8-services/build-tool-detector/app"
+	"github.com/fabric8-services/build-tool-detector/auth/client"
 	"github.com/fabric8-services/build-tool-detector/config"
 	errs "github.com/fabric8-services/build-tool-detector/controllers/error"
 	"github.com/fabric8-services/build-tool-detector/domain/repository"
 	"github.com/fabric8-services/build-tool-detector/domain/repository/github"
 	"github.com/fabric8-services/build-tool-detector/domain/types"
-	"github.com/fabric8-services/build-tool-detector/log"
```

Comment:
I think the formatResponse function will fail since it uses our logger which is hooked into sentry and currently the import has been removed. Can we add this back?